### PR TITLE
[4.0] Fix 3 toolbar buttons

### PR DESCRIPTION
**Pull request by brianteeman**
com-redirect bulk import
com-media create folder
com-media delete

They all had an outline class - hard to see the problem right now except by viewing the source but it will become an issue with the new template being developed

### CODE REVIEW

File: administrator/components/com_redirect/layouts/toolbar/batch.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 $title = $displayData['title'];
 
 ?>
-<button data-toggle="modal" onclick="{jQuery( '#collapseModal' ).modal('show'); return true;}" class="btn btn-outline-primary btn-sm">
+<button data-toggle="modal" onclick="{jQuery( '#collapseModal' ).modal('show'); return true;}" class="btn btn-primary btn-sm">
```
Comment (Quy) [inline administrator/components/com_redirect/layouts/toolbar/batch.php]: For consistency `btn btn-sm`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5e7a5d038386900d296b45a5248e6c4be008b352<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19795">issues.joomla.org/tracker/joomla-cms/19795</a>.</sub>

**Comment by anuragteapot** [comment]
I have tested this item :white_check_mark: successfully on 5e7a5d038386900d296b45a5248e6c4be008b352<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19795">issues.joomla.org/tracker/joomla-cms/19795</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19795">issues.joomla.org/tracker/joomla-cms/19795</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks all

**Pull request merged**